kubernetes-model-generator/kubernetes-model-apiextensions/src/generated/java/io/fabric8/kubernetes/api/model/apiextensions/v1/JSONSchemaProps.java

```diff
@@ -50,10 +50,12 @@
     "example",
     "exclusiveMaximum",
     "exclusiveMinimum",
+    "existingJavaType",
```

Comment:
I haven't realized before that we are changing `JSONSchemaProps` to add those custom extensions.

To stay as close as possible to the k8s representation I think we should not touch this class and rely on an optional [extension](https://swagger.io/docs/specification/openapi-extensions/).
Not sure about the naming, maybe something like:
- `x-javagen-java-type`
- `x-javagen-existing-java-type`

cc. @manusa @rohanKanojia